frontend/styles/partials/_passwordreset.scss

```diff
@@ -1,4 +1,3 @@
-
 .passwordreset {
   background: #F0F2F4;
```

Comment:
can you please check if #F0F2F4 exists in _variables.scss? If so please use the variable
please do the same for other colours in your scss